serverless/src/registration.js

```diff
@@ -1,20 +1,50 @@
 const AWS = require("aws-sdk");
-
+const { httpResponse } = require("./http_response");
 exports.handler = async (event) => {
-  const dbClient = new AWS.DynamoDB.DocumentClient();
-
-  exports.handler = async (event) => {
-    let body;
-    let statusCode = 200;
-    try {
-      body = await dbClient.put(event).promise();
-    } catch (err) {
-      statusCode = err.statusCode;
-      body = err.message;
-    }
-    return {
-      statusCode,
-      body,
-    };
+  const docClient = new AWS.DynamoDB.DocumentClient();
+  let body;
+  let statusCode = 201;
+  let headers = {
+    "Content-Type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Methods": "OPTIONS,POST,GET",
   };
+  let obj = JSON.parse(event.body);
+  const firstName = obj.firstName;
+  const lastName = obj.lastName;
+  const phoneNumber = obj.phoneNumber;
+  const email = obj.email;
+  const street = obj.street;
+  const city = obj.city;
+  const state = obj.state;
+  const zip = obj.zip;
+  const country = obj.country;
+  const dateOfBirth = obj.dateOfBirth;
+  const password = obj.password;
+  const confirmPassword = obj.confirmPassword;
+  const params = {
+    TableName: "next-byte-Registrations-development",
+    Item: {
```

Comment:
You should be able to pass the obj (the body content) as the item here, no need to set up a new object